README.md

```diff
@@ -18,31 +16,62 @@ Visit the full docs [here](https://PrefectHQ.github.io/prefect-openai) to see ad
         <img src="https://img.shields.io/badge/discourse-browse_forum-red.svg?color=0052FF&labelColor=090422&logo=discourse" /></a>
 </p>
 
-## Welcome!
+Visit the full docs [here](https://PrefectHQ.github.io/prefect-openai) to see additional examples and the API reference.
 
-Prefect integrations for working with OpenAI.
+The prefect-dask collection makes it easy to include AI in your flows. Check out the examples below to get started!
```

Comment:
```suggestion
The prefect-openai collection makes it easy to leverage the capabilities of AI in your flows. Check out the examples below to get started!
```

Reply:
Haha silly me, I copied prefect-dask landing page and didn't update the name!